R/utils.R

```diff
@@ -399,3 +399,9 @@ select_decorators <- function(decorators, scope) {
     list()
   }
 }
+
+
+prepare_code <- function(x) {
+  code <- teal.code::get_code(req(x))
+  paste(as.character(styler::style_text(code)), collapse = "\n")
+}
```

Comment:
No styler before the release. Show R code might disappear completely from the modules soon 
```suggestion
prepare_code <- function(x) {
  code <- teal.code::get_code(req(x))
  paste(as.character(styler::style_text(code)), collapse = "\n")
}
```

Reply:
@llrs-roche if that's the case, let's revert this, and let's open new issue to discuss.

Reply:
Done! reverted on [3481365](https://github.com/insightsengineering/teal.modules.general/pull/847/commits/34813656c849928368bf1859e515d3f98a9d4621), issue opened at #853